src/app/application.tsx

```diff
@@ -235,6 +241,10 @@ export class Application {
           },
           this.config.CorsPolicy.TargetOrigins
         );
+      } else if (this.config.CorsPolicy && this.config.CorsPolicy.Embedded) {
+        onInitialized(id, (data: any) => {
```

Comment:
The `onInitialized` parameter is optional, but I don't see any checks before calling it here.